From 0ee8790d45cf00a8524f1d76ab27721af75b4bc2 Mon Sep 17 00:00:00 2001
From: phanhoang1366 <84622359+phanhoang1366@users.noreply.github.com>
Date: Sat, 15 Oct 2022 17:27:48 +0700
Subject: [PATCH] patch

---
 bindings/Python/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bindings/Python/setup.py b/bindings/Python/setup.py
index c333168..8141bcc 100644
--- a/bindings/Python/setup.py
+++ b/bindings/Python/setup.py
@@ -1,6 +1,6 @@
 # coding=utf-8
 from distutils.core import setup, Extension
 
-setup(name='_judger',
+setup(name='judger',
       version='2.1',
       packages=["_judger"])
\ No newline at end of file
-- 
2.37.2.windows.2

